index.js

```diff
@@ -25,13 +26,24 @@ function Level (location, opts) {
   this.location = location
   this.prefix = opts.prefix || DEFAULT_PREFIX
   this.version = parseInt(opts.version || 1, 10)
+
+  // Experimental, do not externally rely on this object yet.
+  // See Level/community#42.
+  this.supports = {
+    bufferKeys: opts.bufferKeys !== false && Level.binaryKeys,
+    permanence: true,
+    snapshots: true, // For now (#86)
+    seek: false, // #178
+    clear: false, // #175
+    createIfMissing: false,
+    errorIfExists: false
+  }
```

Comment:
@ralphtheninja Could manifests really be this simple? :)

Reply:
I guess, unless you need more information, maybe you want to encode which version of something is supported and what not.

Reply:
We can add information as needed. Like `supports.version`, to expose the `abstract-leveldown` version (or `levelup` version... hmm so we'd better merge `abstract-leveldown` and `levelup` before that), or richer info like `supports.methods.approximateSize.promise` (idk, let's only add what we really need).

Reply:
Looking at `suite()` now and this is imo already acceptable. That way ^ is just another way of passing options to `suite()` but in the form of a manifest object on the db instance rather than an options object in the tests.